.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -123,16 +124,11 @@ jobs:
           mkdir Kokkos
           cp -rf ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
 
-      - name: Install Python 3.10
-        uses: actions/setup-python@v5
-        with:
-          python-version: '3.10'
-
```

Comment:
This action usually helps with PATHs etc. You can try to bring it back and see if this solves the problem.
GH native python can be a little unreliable sometimes.